net-proxy/trojan/files/trojan.initd

```diff
@@ -0,0 +1,35 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+TJ_CONFIG="/etc/trojan/config.json"
+TJ_PIDFILE="/run/trojan.pid"
+TJ_LOG="/var/log/trojan.log"
+
+depend() {
+	need net
+}
+
+checkconfig() {
+	if [ ! -f ${TJ_CONFIG} ]; then
+		ewarn "${TJ_CONFIG} does not exist."
```

Comment:
I'm confused. Do you really want it to be only a mild warning, or do you want to fail when it's not present?